command/agent/config.go

```diff
@@ -902,13 +940,32 @@ func isTooManyColons(err error) bool {
 	return err != nil && strings.Contains(err.Error(), tooManyColons)
 }
 
+// Merge is used to merge two ACL configs together
```

Comment:
You might add a comment about precedance- it looks like b takes precedance over a.